docs/index.rst

```diff
@@ -6,17 +6,31 @@
 Welcome to skops's documentation!
 =================================
 
+``skops`` is a Python library helping you share you `scikit-learn
```

Comment:
How about linking `skops` to github here? AFAICT, there is no link to the repo anywhere in the documentation at all.

Reply:
Done!